app/processing/download-and-parse.js

```diff
@@ -0,0 +1,20 @@
+const storage = require('../storage')
```

Comment:
can we standardise what this input is referred to as? some places it's: `storage`, others: `blobStorage` or `config`

similar for `config/message` and `config/processing` -- both of these we have examples of extracting function names from this, i.e. `{ functionName } = require('config/processing')`

should we import the whole config module or specify function names like we do for most modules?

Reply:
Within the application code, I've left/changed `storage` to be consistent and not tied to a particular storage tech.  Whereas the tests, I left it as `blobStorage` because it was also using additional functions direct from the SDK and felt it made more sense.

I think the latter is subjective as it depends what makes it more readable in the context you're using the config.  I think I'd definitely be pushing the scope if I started rewriting all of those too.